src/pages/VideoCall/VideoCallFrame.tsx

```diff
@@ -100,10 +106,7 @@ const VideoCallFrame = ({
                             name="instantvisio-feedback"
                             value={value}
                             precision={1}
-                            onChange={(event, value) => {
-                                setValue(value)
-                                value && addCallRating(value)
-                            }}
+                            onChange={(event, value) => onChange(value)}
```

Comment:
```suggestion
                            onChange={onChange}
```
And in your function `onChange`

```
  const onChange = (_, value) => {
    if (value) {
        setValue(value)
        addCallRating(value)
    }  
  }
```

Reply:
it s ok now in the last push (onchange function updated)